packages/xrpc-server/tests/errors.test.ts

```diff
@@ -45,33 +46,33 @@ describe('Procedures', () => {
         which: 'foo',
       })
       throw new Error('Didnt throw')
-    } catch (e: any) {
+    } catch (e) {
       expect(e instanceof XRPCError).toBeTruthy()
-      expect(e.success).toBeFalsy()
-      expect(e.error).toBe('Foo')
-      expect(e.message).toBe('It was this one!')
+      expect((e as XRPCError).success).toBeFalsy()
+      expect((e as XRPCError).error).toBe('Foo')
+      expect((e as XRPCError).message).toBe('It was this one!')
     }
     try {
       await client.call('io.example.error', {
         which: 'bar',
       })
       throw new Error('Didnt throw')
-    } catch (e: any) {
+    } catch (e) {
       expect(e instanceof XRPCError).toBeTruthy()
-      expect(e.success).toBeFalsy()
-      expect(e.error).toBe('Bar')
-      expect(e.message).toBe('It was that one!')
+      expect((e as XRPCError).success).toBeFalsy()
+      expect((e as XRPCError).error).toBe('Bar')
+      expect((e as XRPCError).message).toBe('It was that one!')
     }
     try {
       await client.call('io.example.error', {
         which: 'other',
       })
       throw new Error('Didnt throw')
-    } catch (e: any) {
+    } catch (e) {
       expect(e instanceof XRPCError).toBeTruthy()
-      expect(e.success).toBeFalsy()
-      expect(e.error).toBe('InvalidRequest')
-      expect(e.message).toBe('Invalid Request')
+      expect((e as XRPCError).success).toBeFalsy()
+      expect((e as XRPCError).error).toBe('InvalidRequest')
+      expect((e as XRPCError).message).toBe('Invalid Request')
```

Comment:
Not suggesting any changes around this here/now, but I realized recently that node's `assert()` hooks nicely into the type system so that these pesky `as XRPCError`s can be removed.  Looks something like this:
```ts
import assert from 'assert'
// ...
      assert(e instanceof XRPCError) // TS now knows e is an XRPCError
      expect(e.success).toBeFalsy()
```

Reply:
Nice, good to know